apps/marketplace/components/BuyerATM/BuyerATMAdditionalInformationStage.js

```diff
@@ -2,21 +2,53 @@ import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import { actions, Form } from 'react-redux-form'
-import { required, validPhoneNumber, dateIs2DaysInFuture } from 'marketplace/components/validators'
 import formProps from 'shared/form/formPropsSelector'
 import Textfield from 'shared/form/Textfield'
-import AUheading from '@gov.au/headings/lib/js/react.js'
+import AUheadings from '@gov.au/headings/lib/js/react.js'
+import format from 'date-fns/format'
+import addDays from 'date-fns/add_days'
+import {
+  required,
+  validPhoneNumber,
+  validDate,
+  dateIs2DaysInFuture,
+  dateIsBefore
+} from 'marketplace/components/validators'
 import ErrorAlert from 'marketplace/components/Alerts/ErrorAlert'
-import ClosingDateControl from 'marketplace/components/BuyerBriefFlow/ClosingDateControl'
+import DateControl from 'marketplace/components/BuyerBriefFlow/DateControl'
 import styles from './BuyerATMAdditionalInformationStage.scss'
 
-class BuyerATMAdditionalInformationStage extends Component {
+const requiredContactNumber = v => required(v.contactNumber)
+const contactNumberFormat = v => validPhoneNumber(v.contactNumber)
+const requiredClosedAt = v => required(v.closedAt)
+const closedAtIsValid = v => validDate(v.closedAt)
+const closedAtIs2DaysInFuture = v => !closedAtIsValid(v) || dateIs2DaysInFuture(v.closedAt)
+const closedAtIsBefore = v => !closedAtIsValid(v) || dateIsBefore(v.closedAt, addDays(new Date(), 364))
+
+export const done = v =>
+  requiredContactNumber(v) &&
+  contactNumberFormat(v) &&
+  requiredClosedAt(v) &&
+  closedAtIsValid(v) &&
+  closedAtIs2DaysInFuture(v) &&
+  closedAtIsBefore(v)
+
+export class BuyerATMAdditionalInformationStage extends Component {
   constructor(props) {
     super(props)
-
+    this.state = {
+      fileCount: 1
+    }
```

Comment:
I can't see any references to this.  Why has this been added?

Reply:
removed from code